pynestml/codegeneration/nest_code_generator_utils.py

```diff
@@ -81,59 +80,56 @@ def generate_code_for(cls,
         # generate temporary install directory
         install_path = tempfile.mkdtemp(prefix="nestml_target_")
 
-        # generate unique ID
-        if uniq_id is None:
-            uniq_id = str(uuid.uuid4().hex)
-
         # read neuron model from file?
-        if not "\n" in nestml_neuron_model and ".nestml" in nestml_neuron_model:
+        if "\n" not in nestml_neuron_model and ".nestml" in nestml_neuron_model:
             with open(nestml_neuron_model, "r") as nestml_model_file:
                 nestml_neuron_model = nestml_model_file.read()
 
         # update neuron model name inside the file
-        neuron_model_name_orig = re.findall(r"neuron\ [^:\s]*:", nestml_neuron_model)[0][7:-1]
-        neuron_model_name_uniq = neuron_model_name_orig + uniq_id
-        nestml_model = re.sub(r"neuron\ [^:\s]*:",
-                              "neuron " + neuron_model_name_uniq + ":", nestml_neuron_model)
-        neuron_uniq_fn = neuron_model_name_uniq + ".nestml"
-        with open(neuron_uniq_fn, "w") as f:
+        neuron_model_name = re.findall(r"neuron [^:\s]*:", nestml_neuron_model)[0][7:-1]
```

Comment:
lines 89-93 are no longer necessary because we're not appending the uuid any longer, right?

Reply:
I have removed the string substitution but have retained extracting the neuron name since we return the neuron name from this function.